### ISSUE TITLE
error X3535: GatherGreen requires shader model 5 or higher

### ISSUE THREAD

**Issue by Coliostro** [Blinue/Magpie#238]
### Magpie version 程序版本

0.7.1.0

### Windows version 系统版本

10.0.19042

### Log files 日志文件

[Magpie.log](https://github.com/Blinue/Magpie/files/7693473/Magpie.log)
[Runtime.log](https://github.com/Blinue/Magpie/files/7693474/Runtime.log)

**Comment by Blinue** [comment]
In fact Magpie's FSR does not support DirectX feature level 10. I'll fix it in v0.8 #225

**Comment by Blinue** [comment]
Use this instead.

``` json
{
    "name": "FSR",
    "effects": [
        {
            "effect": "FSR_EASU_DX10",
            "scale": [ -1, -1 ]
        },
        {
            "effect": "FSR_RCAS",
            "sharpness": 0.87
        }
    ]
}
```
